### ISSUE TITLE
Cannot build 0.1.105 on FreeBSD

### ISSUE THREAD

**Issue by probonopd** [vasi/squashfuse#108]
squashfuse 0.1.104 is in FreeBSD Ports. But I need 0.1.105 so that I can use `ll.h`. So I edited the Makefile a bit:

```
PORTNAME=       squashfuse
PORTVERSION=    0.1.105
CATEGORIES=     sysutils
MASTER_SITES=   ${MASTER_SITE_GITHUB}/vasi/${PORTNAME}/archive/refs/tags/
DISTNAME=       ${PORTVERSION}
EXTRACT_SUFX=   .tar.gz
PKGNAMEPREFIX=  fusefs-

MAINTAINER=     jsmith@resonatingmedia.com
COMMENT=        Mount a squashfs archive and treat it like a local file system
WWW=            https://github.com/vasi/squashfuse

LICENSE=        BSD2CLAUSE
LICENSE_FILE=   ${WRKSRC}/LICENSE

# Specify the direct URL to the tarball
DISTFILES=      ${DISTNAME}${EXTRACT_SUFX}

USES=           autoreconf fuse libtool pkgconfig
GNU_CONFIGURE=  yes

INSTALL_TARGET= install-strip

# Set the correct path to the extracted source code directory
WRKSRC=         ${WRKDIR}/${PORTNAME}-${PORTVERSION}

.include <bsd.port.mk>
```

But the build fails with:

```
--- squashfuse_ll-ll_main.o ---
In file included from ll_main.c:25:
--- squashfuse-hl.o ---
In file included from hl.c:26:
./fuseprivate.h:31:10: fatal error: 'fuse3/fuse.h' file not found
#include <fuse3/fuse.h>
         ^~~~~~~~~~~~~~
--- squashfuse_ll-ll_main.o ---
./ll.h:31:10: fatal error: 'fuse3/fuse_lowlevel.h' file not found
#include <fuse3/fuse_lowlevel.h>
         ^~~~~~~~~~~~~~~~~~~~~~~
--- squashfuse_extract-nonstd-makedev.o ---
```

There is no fuse3 in FreeBSD:

```
# pkg list fusefs-libs
/usr/local/include/fuse.h
/usr/local/include/fuse/cuse_lowlevel.h
/usr/local/include/fuse/fuse.h
/usr/local/include/fuse/fuse_common.h
/usr/local/include/fuse/fuse_common_compat.h
/usr/local/include/fuse/fuse_compat.h
/usr/local/include/fuse/fuse_kernel.h
/usr/local/include/fuse/fuse_lowlevel.h
/usr/local/include/fuse/fuse_lowlevel_compat.h
/usr/local/include/fuse/fuse_opt.h
/usr/local/include/ulockmgr.h
/usr/local/lib/libfuse.a
/usr/local/lib/libfuse.so
/usr/local/lib/libfuse.so.2
/usr/local/lib/libfuse.so.2.9.9
/usr/local/lib/libulockmgr.a
/usr/local/lib/libulockmgr.so
/usr/local/lib/libulockmgr.so.1
/usr/local/lib/libulockmgr.so.1.0.1
/usr/local/libdata/pkgconfig/fuse.pc
/usr/local/share/doc/fusefs/libs/how-fuse-works
/usr/local/share/doc/fusefs/libs/kernel.txt
/usr/local/share/licenses/fusefs-libs-2.9.9_2/LGPL21
/usr/local/share/licenses/fusefs-libs-2.9.9_2/LICENSE
/usr/local/share/licenses/fusefs-libs-2.9.9_2/catalog.mk
```

Is it true that from 0.1.104 to 0.1.105 libfuse3 became a requirement? This would be a bit unexpected for a patch level release.

cc @Jehops

**Comment by DrDaveD** [comment]
I have only been the release manager for about a month.  I think the previous release manager didn't pay enough attention to semantic versioning, you're right on that.  I have been updating the minor version number since then for each new significant functionality change. However there doesn't seem to be an obvious change in 0.1.105 from about a year ago affecting fuse3.  Support for fuse3 was added in 0.1.104 about two years ago; it seems more likely that that is when the problem was introduced.

**Comment by vasi** [comment]
So FreeBSD _does_ have fuse 3, packaged as "fusefs-libs3". But it looks like `pkg-config --cflags fuse3` on both BSD/Linux returns a path such that "fuse.h" works, rather than a path that requires fuse3/fuse.h

**Comment by vasi** [comment]
We probably should also not add fuse_lowlevel.h here, since some systems won't have that!

**Comment by probonopd** [comment]
Thank you very much. So, which version will be buildable on FreeBSD?

**Comment by DrDaveD** [comment]
Assuming that really did fix it, it should be 0.3.1.  Can you please test the main branch to see if it solves the problem?
